### PR TITLE
Hide cmd output from users and some improvements

### PR DESCRIPTION
**Pull request by MaxymVlasov**

### CODE REVIEW

**Comment by voatsap** [comment]
Don't get if it works with terraform apply output suppressions. 
Can you please show real cluster creation action logs as test output?

**Comment by MaxymVlasov** [comment]
@voatsap 
VERBOSE_LVL=DEBUG
https://github.com/shalb/cluster.dev/runs/502180123

VERBOSE_LVL=INFO
https://github.com/shalb/cluster.dev/runs/502234457

VERBOSE_LVL=INFO + demo of "function forks too long"
https://github.com/shalb/cluster.dev/runs/502243083

Simple Error demo
https://github.com/shalb/cluster.dev/runs/502246668

File: entrypoint.sh [hunk 1]
```diff
@@ -11,6 +11,69 @@ readonly CLOUD_PASS=$3
...
+            break
+        fi
+        sleep 1
+        ((seconds++))
+    done
+
+    wait $proc_pid
+    exit_code=$?
+    # `> /dev/null 2>&1 &` and `wait` need to suppress "Terminated' message
+    kill $tail_pid > /dev/null 2>&1 &
```
Comment (romanprog) [inline entrypoint.sh]: is variable $tail_pid have correct value outside "until" loop? 
(https://stackoverflow.com/questions/16854280/a-variable-modified-inside-a-while-loop-is-not-remembered)

Comment (MaxymVlasov) [reply entrypoint.sh]: Yes. It always try to terminate PID or "" and when I tesst before suppress stdout - it always terminate `tail`

Just recheck and all seems good

```bash
$ export VERBOSE_LVL=NOTICE && rm -rf terraform/aws/backend/.terraform && bash cmd.sh
+ cd terraform/aws/backend
+ terraform init
13:22:18 - Command 'cd terraform/aws/backend && terraform init' executing more than 4s. Start print out exist and future output for this command.
tail_pid inside loop: 4088

Initializing the backend...

Initializing provider plugins...
- Checking for available provider plugins...
- Downloading plugin for provider "aws" (hashicorp/aws) 2.52.0...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
tail_pid outside loop: 4088
```

**Pull request merged**